### ISSUE TITLE
Defect with one-line-tall terminal and about box (re: PR#5599)

### ISSUE THREAD

**Issue by fishzle** [microsoft/terminal#5808]
# Environment

```none
Windows build number: 10.0.19041.0
Windows Terminal version (if applicable): 0.11.1251.0

Any other software?
I'm using focus follows mouse.
```

# Steps to reproduce
make a one-line-tall terminal by resizing vertically.
Click the down arrow and select About.
You can see 2 lines (that's ok)

![image](https://user-images.githubusercontent.com/8211862/81409339-4c4b4a00-9182-11ea-983a-8ff3cbd909e7.png)

If you grab the left edge of the terminal and resize it wider. It gives an unexpected result.
(where's my text? my cursor? where's the about box? where's the OK button?)
![image](https://user-images.githubusercontent.com/8211862/81409383-59683900-9182-11ea-9ffd-68e053b9daad.png)

similar but slightly different issue in a 2-line-tall terminal
![image](https://user-images.githubusercontent.com/8211862/81410405-03949080-9184-11ea-9835-da8edd826fae.png)

You can press ESC and it closes the About box. Mostly.

Sometimes when the mouse is in the correct area, the focus doesn't go to the right place and it takes a few attempts of wiggling the mouse in/out of the window so that ESC does it's magic.

When the focus isn't right, ESC doesn't do anything - which is ok. However, when it's in this state I #found I could not close wt from the icon bar either. I couldn't right click and "Close window"

Loss-of-focus issue hard to reproduce.

# Expected behavior
1. Resizing the terminal using left-edge, right-edge, bottom-edge, top-edge should redraw or cancel the about box.
2. Always able to right-click to close from iconbar.

# Actual behavior

**Comment by DHowett-MSFT** [comment]
We somewhat decided this was a “play stupid games, win stupid prizes” sort of problem; it’s something we should fix, but not above priority 3 :p

**Comment by zadjii-msft** [comment]
You know what, this is just going to be fixed by #2380, said to be caused by microsoft/microsoft-ui-xaml#3577.

/dup  #2380
